.github/actions/setup-wp-env/action.yml

```diff
@@ -0,0 +1,53 @@
+name: Setup WP Env
+description: Setup WordPress environment for testing
+
+inputs:
+    wp:
+        description: WordPress version
+        required: true
+    php:
+        description: PHP version
+        required: true
```

Comment:
don't you allow empty and then it'll use `null`?

Reply:
Done